ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -213,13 +213,24 @@ data:
       - labeldrop:
         - filename
         - stream
-      - pack:
-          labels:
-          - boot_id
-          - systemd_unit
-          - host
+      - match:
+          # To get labels for a new systemd_unit exclude it by adding it in the selector here and include
+          # it by adding it in the selector below.  For any systemd_units, besides these, we will pack
+          # (i.e., no label) to avoid high cardinality.
+          selector: '{systemd_unit!~"auditd.service|crio.service|kubelet.service|NetworkManager.service|ovs-vswitchd.service|ovs-configuration.service|ovsdb-server.service"}'
+          stages:
+            - pack:
+                labels:
+                - boot_id
+                - systemd_unit
+                - host
+      - match:
+          # These systemd_units will get a systemd_unit label; if you add one, be sure to monitor number of
+          # Active Streams in Loki Dashboard to avoid over burdening our instance of Promtail/Loki.
+          selector: '{systemd_unit=~"auditd.service|crio.service|kubelet.service|NetworkManager.service|ovs-vswitchd.service|ovs-configuration.service|ovsdb-server.service"}'
```

Comment:
I believe this will mean that systemd_unit=auditd.service will now no longer have boot_id or host "packed" into the json. It might not be packed at all actually. 

You need a parallel pack stage just WITHOUT systemd_unit, see the namespace example above.

Reply:
I added stages and a `pack` section below these matches; hopefully that will do it.

Reply:
I think it reads as: "if you don't match specific systemd_units, we pack all 3; if you match, we pack only 2; later the systemd_unit is added as a label".

Reply:
Is this tested and working as expected?

Reply:
Looking at my [rehearsal job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40773/rehearse-40773-pull-ci-openshift-machine-api-provider-gcp-release-4.14-e2e-gcp-ovn/1674366681868144640), this [loki query](https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%7B%22datasource%22:%22PCEB727DF2F34084E%22,%22queries%22:%5B%7B%22expr%22:%22%7Binvoker%3D%5C%22openshift-internal-ci%2Frehearse-40773-pull-ci-openshift-machine-api-provider-gcp-release-4.14-e2e-gcp-ovn%2F1674366681868144640%5C%22,%5Cn%20%20type%3D%5C%22journal%5C%22,%5Cn%20%20systemd_unit%3D~%5C%22auditd.service%5C%22%7D%5Cn%5Cn%7Cunpack%7C%20line_format%20%5C%22%7B%7B.boot_id%7D%7D%5C%22%22,%22refId%22:%22A%22,%22editorMode%22:%22code%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%221688035246000%22,%22to%22:%221688043106000%22%7D%7D):

```
{invoker="openshift-internal-ci/rehearse-40773-pull-ci-openshift-machine-api-provider-gcp-release-4.14-e2e-gcp-ovn/1674366681868144640",
  type="journal",
  systemd_unit=~"auditd.service"}
|unpack| line_format "{{.boot_id}}"
```

shows `boot_id` values like this:

```
2023-06-29 07:13:44	4c39df1811de48aeaab084a78a74cc0c
2023-06-29 07:13:44	4c39df1811de48aeaab084a78a74cc0c
2023-06-29 07:13:44	4c39df1811de48aeaab084a78a74cc0c
```

I think this shows that the systemd_unit is now a label and also `boot_id` is packed.

Picking some other [recent job](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1678163771828211712) that doesn't have this new configuration, if you use a [query that includes the `systemd_unit` label](https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%7B%22datasource%22:%22PCEB727DF2F34084E%22,%22queries%22:%5B%7B%22expr%22:%22%7Binvoker%3D%5C%22openshift-internal-ci%2Fperiodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade%2F1678163771828211712%5C%22,%20type%3D%5C%22journal%5C%22,%20systemd_unit%3D%5C%22NetworkManager.service%5C%22%7D%22,%22refId%22:%22A%22,%22editorMode%22:%22code%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%221688941536556%22,%22to%22:%221688952677354%22%7D%7D):

```
{invoker="openshift-internal-ci/periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-gcp-ovn-rt-upgrade/1678163771828211712",
   type="journal",
   systemd_unit="NetworkManager.service"}
```

you will not see any logs.